install.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+print_usage() {
+    cat <<EOF
+Usage: install.sh [options]
+
+Options:
+  --root /path/to/root     alternative install root (default /)
+  --prefix /prefix         directory prefix (default /usr)
+  --etcdir /etc            specify etc directory path (default /etc)
+  --nonroot                install Scylla without required root priviledge
+  --help                   this helpful message
+EOF
+    exit 1
+}
+
+root=/
+etcdir=/etc
+nonroot=false
+
+while [ $# -gt 0 ]; do
+    case "$1" in
+        "--root")
+            root="$2"
+            shift 2
+            ;;
+        "--prefix")
+            prefix="$2"
+            shift 2
+            ;;
+        "--etcdir")
+            etcdir="$2"
+            shift 2
+            ;;
+        "--nonroot")
+            nonroot=true
+            shift 1
+            ;;
+        "--help")
+            shift 1
+	    print_usage
+            ;;
+        *)
+            print_usage
+            ;;
+    esac
+done
+
+if [ -z "$prefix" ]; then
+    if $nonroot; then
+        prefix=~/scylladb
+    else
+        prefix=/opt/scylladb
+    fi
+fi
+
+scylla_version=$(cat SCYLLA-VERSION-FILE)
+scylla_release=$(cat SCYLLA-RELEASE-FILE)
+
+rprefix=$(realpath -m "$root/$prefix")
+if ! $nonroot; then
+    retc="$root/$etcdir"
+    rusr="$root/usr"
+else
+    retc="$rprefix/$etcdir"
+fi
+
+install -d -m755 "$rprefix"/share/cassandra/bin
+if ! $nonroot; then
+    install -d -m755 "$rusr"/bin
+fi
+
+thunk="#!/usr/bin/env bash
```

Comment:
We should use relocatable python3 to run ALL our python script.
Also, we should create symlink on /usr/bin/cqlsh, instead of writing thunk to /usr/bin/cqlsh.
Because we want to use relocatable python3 both on /opt/scylladb/share/cassandra/bin/cqlsh and /usr/bin/cqlsh.


We need following fix:
https://github.com/syuu1228/scylla-cqlsh/commit/4ba3c7a4a2e31a9a13903e8ea9a4ac63459a8c67

Reply:
took this change, but still this package isn't depended on the relocatable python3, can be installed on it's own and would work without relocatable python3